Makefile

```diff
@@ -51,7 +51,8 @@ clean-test: ## remove test and coverage artifacts
 	rm -fr .pytest_cache
 
 lint: ## check style with flake8
-	flake8 rev_ai tests
+	flake8 rev_ai tests --ignore=F401,W504,E731
+	flake8 rev_ai src --ignore=F401,W504
```

Comment:
Why no E731?

Reply:
https://lintlyci.github.io/Flake8Rules/rules/E731.html

This makes some sense in a public api where everything needs to be highly readable. Assigning lambdas is a nice feature to have for testing though